cmd/argo/commands/clustertemplate/list.go

```diff
@@ -57,14 +54,3 @@ func NewListCommand() *cobra.Command {
 	command.Flags().StringVarP(&listArgs.output, "output", "o", "", "Output format. One of: wide|name")
 	return command
 }
-
-func printTable(wfList []wfv1.ClusterWorkflowTemplate) {
```

Comment:
`printTable` is actually _only_ used inside of the respective `list.go` files, so it shouldn't be moved. 

If you did a regex search on `print*`, that's my bad, I didn't realize this `printTable` function would be included in that search.
I only [meant](https://github.com/argoproj/argo-workflows/pull/12803#discussion_r1536717614) the `print<Resource>` functions, like `printClusterWorkflowTemplate*` (and the ones for `cron` and `template` commands, respectively).
`print<Resource>` and `unmarshal<Resource>` are used in multiple files, so they should really be defined in a common helper file like `util.go`. `printTable` is only used in one file, so it doesn't match them.

Reply:
OK, I modified it, thanks！